Tests/Logic/ObjectMapping/RKPaginatorTest.m

```diff
@@ -474,6 +482,18 @@ - (void) testLoadingAPageWithCustomPaginator
     [paginator waitUntilFinished];
 }
 
+- (void) testLoadingAPageWithCustomPaginatorContainingParameter
+{
+    RKObjectManager* manager = [RKTestFactory objectManager];
+    [RKObjectManager setSharedManager:manager];
+    
+    manager.paginationMapping = [self customPaginationMapping];
+    RKPaginator* paginator = [manager paginatorWithPathPattern:RKPaginatorTestResourcePathPattern parameters:@{@"param1":@"value1"}];
+    [paginator loadPage:1];
+    [paginator waitUntilFinished];
+    expect(paginator.URL.relativeString).to.contain(@"param1=value1");
```

Comment:
could you also just make sure loading a second page preserves the parameters? Thanks!